modules/cleanup/cleanup.go

```diff
@@ -0,0 +1,143 @@
+package cleanup
+
+import (
+	"embed"
+	"fmt"
+	"strings"
+
+	"github.com/GoMudEngine/GoMud/internal/buffs"
+	"github.com/GoMudEngine/GoMud/internal/events"
+	"github.com/GoMudEngine/GoMud/internal/plugins"
+	"github.com/GoMudEngine/GoMud/internal/rooms"
+	"github.com/GoMudEngine/GoMud/internal/users"
+	"github.com/GoMudEngine/GoMud/internal/util"
+)
+
+var (
+
+	//////////////////////////////////////////////////////////////////////
+	// NOTE: The below //go:embed directive is important!
+	// It embeds the relative path into the var below it.
+	//////////////////////////////////////////////////////////////////////
+
+	//go:embed files/*
+	files embed.FS
+)
+
+// ////////////////////////////////////////////////////////////////////
+// NOTE: The init function in Go is a special function that is
+// automatically executed before the main function within a package.
+// It is used to initialize variables, set up configurations, or
+// perform any other setup tasks that need to be done before the
+// program starts running.
+// ////////////////////////////////////////////////////////////////////
+func init() {
+	//
+	// We can use all functions only, but this demonstrates
+	// how to use a struct
+	//
+	c := CleanupModule{
+		plug: plugins.New(`cleanup`, `1.0`),
+	}
+
+	//
+	// Add the embedded filesystem
+	//
+	if err := c.plug.AttachFileSystem(files); err != nil {
+		panic(err)
+	}
+	//
+	// Register any user/mob commands
+	//
+	c.plug.AddUserCommand(`trash`, c.trashCommand, false, false)
+	c.plug.AddUserCommand(`bury`, c.buryCommand, false, false)
```

Comment:
There are also`mobcommands` for `bury` and `trash` that need to be converted. mob commands vary slightly in some cases, but in something simple like this, they typically just don't send the mob a message, and no xp rewards etc.

https://github.com/GoMudEngine/GoMud/blob/master/internal/mobcommands/mobcommands.go#L32

https://github.com/GoMudEngine/GoMud/blob/master/internal/mobcommands/mobcommands.go#L66

https://github.com/GoMudEngine/GoMud/blob/master/internal/mobcommands/bury.go

https://github.com/GoMudEngine/GoMud/blob/master/internal/mobcommands/trash.go